runtime/drivers/postgres/postgres.go

```diff
@@ -15,6 +15,37 @@ import (
 
 func init() {
 	drivers.Register("postgres", driver{})
+	drivers.RegisterAsConnector("postgres", driver{})
+}
+
+var spec = drivers.Spec{
+	DisplayName: "Postgres",
+	Description: "Connect to Postgres.",
+	SourceProperties: []drivers.PropertySchema{
+		{
+			Key:         "sql",
+			Type:        drivers.StringPropertyType,
+			Required:    true,
+			DisplayName: "SQL",
+			Description: "Query to extract data from Postgres.",
+			Placeholder: "select * from table;",
+		},
+		{
+			Key:         "pg_database_url",
+			DisplayName: "Postgress Connection String",
+			Type:        drivers.StringPropertyType,
+			Required:    false,
+			Href:        "https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING",
```

Comment:
Since it's a "connection URI", not "connection string", should it link to `https://www.postgresql.org/docs/current/libpq-connect.html#id-1.7.3.8.3.6` instead?

Reply:
It accepts both types. Even string format `host=localhost port=5432 dbname=postgres user=postgres password=postgres` works.